core/templates/dev/head/pages/base.html

```diff
@@ -198,6 +198,8 @@
     {% include 'components/create_button/create_activity_button_directive.html' %}
 
     <script src="{{TEMPLATE_DIR_PREFIX}}/app.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/services/stateful/BackgroundMaskService.js"></script>
+    <script src="{{TEMPLATE_DIR_PREFIX}}/services/stateful/FocusManagerService.js"></script>
```

Comment:
Same, if moving these two next to all services import won't break anything, then could you please move it ?